### PR TITLE
Fix readme

### LINKED ISSUE #125: Fix backend documentation
# User Story
Fix backend documentation
# Acceptance Criteria
- Fix scarb version 
- Remove chatgpt code parts
- More simple, remove unnecessary parts

### PR DESCRIPTION
**Pull request by EmmanuelAR**
# Pull Request

- [x] Closes #125 
- [x] Added tests (if necessary)
- [x] Run tests
- [x] Run formatting
- [x] Commented the code

## Changes description

- Fix backend readme, remove chatgpt generic documentation. Only setup env info 
- Add Daniel Garbanzo in main readme.
- Remove uml from main readme

## Current output

Backend readme:
Captura de pantalla 2024-10-03 a la(s) 4 41 24 p  m

Main readme:
Captura de pantalla 2024-10-03 a la(s) 4 42 25 p  m

### CODE REVIEW

File: contracts/README.md [hunk 1]
```diff
@@ -8,112 +8,48 @@
...
 
-To run the unit tests for the contracts, run the following command:
-
-```bash
-snforge test
+    To run the unit tests for the contracts, run the following command:
 
-```
+    ```bash
+    snforge test
```
Comment (bitfalt) [inline contracts/README.md:42]: Change the command to scarb run test or add another line explaining that scarb run test can also run.

**Review by bitfalt — CHANGES_REQUESTED** [review]

File: contracts/README.md [hunk 2]
```diff
@@ -38,6 +38,10 @@
 
     To run the unit tests for the contracts, run the following command:
 
+    ```bash
+    scarb test
```
Comment (bitfalt) [inline contracts/README.md:42]: correct command would be `scarb run test`. Only fix missing!

Comment (EmmanuelAR) [reply contracts/README.md]: Ready!

**Review by bitfalt — APPROVED** [review]
nice job! :D

**Pull request merged**